chroma_api/stratagem.py

```diff
@@ -64,7 +64,7 @@ def check_duration(duration_key, bundle):
             ),
         }
 
-    if duration < 0:
+    if duration is not None and duration < 0:
```

Comment:
Shouldn't this just return early at the top of this fn if duration is None?

Reply:
sure, i'll move the condition up.